app/main/add_a_new_provider/views.py

```diff
@@ -43,6 +43,14 @@ def form_valid(self, form):
         return super().form_valid(form)
 
 
+class AdvocateDetailsFormView(BaseFormView):
+    def form_valid(self, form):
+        session["solicitor_advocate"] = form.data.get("solicitor_advocate")
```

Comment:
Let's talk about safety of this just to check what's coming through. Are these validated?